### DISCUSSION TITLE
Hi, is shadcn compatible with go_router? I've tried using it but it just crashes haha

### DISCUSSION THREAD

**Discussion by luisomdev** [nank1ro/flutter-shadcn-ui#161] [Q&A] [answered]
I just tried this and wanted to know if it is compatible or if it is simply a bad implementation on my part. By the way, I am using ShadApp.materialRouter.

**Comment by nank1ro** [comment, marked as answer]
Yes it is compatible with any router.
Please share the stack trace I could help you find the reason.
If you think it's a bug in this library open an issue, if it's not I will close it as invalid.

**Reply by luisomdev** [reply]
I've fixed it now, it was a poor implementation on my part. Sorry for wasting your time.
